app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
```

Comment:
- [ ] When trying to sign up, I am getting the following error, kindly fix this issue.

![image](https://user-images.githubusercontent.com/81932478/194514131-678802a0-0ff7-4d0a-9e05-a09d32e67d51.png)

Reply:
@Munsa1 Thank you for your feedback. Actually, I only tested using the seeded user for the login. My bad. Let me fix it.